src/main.c

```diff
@@ -2174,6 +2180,114 @@ void on_middle_click() {
     }
 }
 
+/// As a part of [Issue #41]: https://github.com/Team-10-But-Better/Craft/issues/41
+/// To assist the parseConfigToPrintKeybinds() function, this function looks at a one line of the config.h function
+/// and determines if the line is a #define statement defining a Craft keybind.
+int isDefinedCraftKey(char* str, int start, int end)
+{
+    char str2[end - start];
+    for (int i = 0; i < (end - start); i++)
+    {
+        str2[i] = str[start + i];
+    }
+    int tru = 1;
+    if (str2[0] != 'C')
+        tru = 0;
+    if (str2[1] != 'R')
+        tru = 0;
+    if (str2[2] != 'A')
+        tru = 0;
+    if (str2[3] != 'F')
+        tru = 0;
+    if (str2[4] != 'T')
+        tru = 0;
```

Comment:
Instead of using multiple if statements, check every condition in single if statement, like below:

if(str2[0] != 'C' || str2[1] != 'R' || str2[2] != 'A' || str2[3] != 'F' ||  str2[4] != 'T'){
tru = 0;
{